### PR TITLE
Feat: Add schema library section

### PR DESCRIPTION
**Pull request by iagormoraes**
[Ticket](https://trello.com/c/sp38fMZi/5475-load-and-display-json-schemas-in-the-docs)

## Summary
Now the docs has the new section of schema library, which you can search and see more details about it in a more clear and easy way, also it allows you to copy to clipboard the json schema.

https://github.com/UnumID/UnumID.github.io/assets/13892132/1f239f4e-21e3-48cc-ab28-4498e40fad34

## Changes
Add new section to data schemas page.

## Testing
Tested locally.

## Checklist

- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas _to a borderline excessive amount_
- [ ] I have made any relevant corresponding changes to the documentation including the project readme.
- [x] I have run and tested the changes locally
- [ ] If it is a core feature, I have added an appropriate amount of unit tests.
- [ ] Any dependent changes have been merged and published in upstream projects

### CODE REVIEW

File: src/components/Schemas/Schemas.jsx [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+    </motion.div>
+  );
+}
+
+export default function Schemas(props) {
+  const [isLoading, setIsloading] = useState(true);
+  const [schemas, setSchemas] = useState(null);
+
+  // Numbers of schema is for adding the number of renders for SchemaCardSkeleton to have a good feedback for user.
+  const NUMBER_OF_SCHEMAS = 28;
```
Comment (rsmets) [inline src/components/Schemas/Schemas.jsx:157]: Why are setting this as a constant? Is it a UI decision so only 28 tiles show up in the component? I feel like probably want to show all of them and allow for scrolling? No idea how difficult that would be though.

File: src/components/Schemas/components/SchemaDetails.jsx [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+
+export function SchemaDetails({ schema, onRequestClose }) {
+  const [copied, setCopied] = useState(false);
+  const [isLoading, setIsLoading] = useState(true);
+  const [schemaWithAttributes, setSchemaWithAttributes] = useState(schema);
+
+  const keys = keysLiteral(schema);
+
+  // Copy the schema as it is for easy management of data from user perspective.
+  const handleCopySchema = () => {
```
Comment (rsmets) [inline src/components/Schemas/components/SchemaDetails.jsx:148]: Nice touch

File: docs/schema.mdx [hunk 3]
```diff
@@ -140,3 +141,9 @@ const credentialsList: Credentials = [
 }
 
 ```
+
+## Schema Library
```
Comment (rsmets) [inline docs/schema.mdx:145]: I would say probably should be featured higher on the page. I would say just under Introduction would be best. Definitely want this to be front and center and the can scroll down for the more wordy explanation. But at the very least this ought to be above Example I think.

**Comment by rsmets** [comment]
This is a killer update though Iagor!! Super slick.

File: src/components/Schemas/Schemas.jsx (hunk 1 above)
Comment (iagormoraes) [reply src/components/Schemas/Schemas.jsx:157]: > Why are setting this as a constant? Is it a UI decision so only 28 tiles show up in the component? I feel like probably want to show all of them and allow for scrolling? No idea how difficult that would be though.

This is just for loading the skeletons, we use what is on `schemas` state.

File: docs/schema.mdx (hunk 3 above)
Comment (iagormoraes) [reply docs/schema.mdx]: Indeed it looks better above example section.

File: docs/schema.mdx [hunk 4]
```diff
@@ -103,6 +104,12 @@ You can instead just use the `/jsonSchema` path to only get that information, e.
 Note that each attribute has a `description` and `examples` to help you understand the credential's schema. If you need more detailed information, look at our open source [schema-sdk](https://github.com/UnumID/schema-sdk) [formats](https://github.com/UnumID/schema-sdk/blob/main/src/formats.ts).
 :::
 
+## Schema Library
+
+Next, you can search or select for a schema JSON, see it's properties and attribute, and also copy the schema structure of choice:
```
Comment (jacobleesinger) [inline docs/schema.mdx:109]:
```suggestion
Search for or select a schema, see its properties and attributes, and copy the schema JSON:
```

**Review by jacobleesinger — COMMENTED** [review]
Nice work Iagor! I just have a small suggestion for the text describing the schema library.

**Review by rsmets — APPROVED** [review]
Not merging. Will leave until Jacob's approval.

**Comment by rsmets** [comment]
Maybe worth putting `Schema Definitions` on same heading level as Introduction? May have been a typo why it was a subheading in the first place. 
image

**Comment by rsmets** [comment]
also maybe worth not displaying the `Attribute` field because not super relevant to the customers (really just for our web wallet client). 
image

**Comment by iagormoraes** [comment]
@rsmets You are right, the sections are different, now they are in the same level. In regards to the attribute section, it got removed. 👍

**Review by jacobleesinger — APPROVED** [review]
👍

**Comment by jacobleesinger** [comment]
Not going to merge right now because it looks like you might still be addressing Ray's comments?

**Comment by iagormoraes** [comment]
Right, let's wait his green light.

**Review by rsmets — APPROVED** [review]

**Pull request merged**